src/f64.rs

```diff
@@ -151,7 +146,7 @@ impl Float for F64 {
     }
 
     fn to_i32(&self, rnd: RoundingMode, exact: bool) -> i32 {
-        let ret = unsafe { softfloat_sys::f64_to_i32(self.0, rnd.to_softfloat(), exact) };
+        let ret = unsafe { softfloat_sys::f64_to_i64(self.0, rnd.to_softfloat(), exact) };
```

Comment:
Why?

Reply:
bold hack around c99 types mismatch. fixed to correct type